docs/configuration/OVERVIEW.md

```diff
@@ -36,103 +45,153 @@ A component will have to handle different types of configurations.
+> [!WARNING]
+> The field name `id` should not be used in the configuration, as we use this field in our internal implementation.
 
-### Configuration coming from config store
+### Customized configuration
 
-- Each __component type__ can have a customized config (compliant with the store model). This custom config can be
-  asynchronously loaded from a backend server, or it can be injected in the body tag of the app. In both cases, this
-  content will be used to update the store.
-- If present, it will override the default config. (Don't worry about the implementation now, we'll see it later)
+- Each __configurable component__ can have a customized configuration, which can be asynchronously loaded from a backend server,
+  or it can be injected in the body tag of the app.
+- If present, it will override the default configuration. Don't worry about the implementation now, we'll see it later.
 
-### Global config
+### Global configuration
 
-- The common configuration is the one used in multiple components (it can be a date format, price display, type of input
-  form fields from an Angular material input element ...)
-- A common configuration is defined in every library. The application common configuration (__global config__) will be
-  the result of the __merge of all common__ configurations.
+- The __common configuration__ is the one used in multiple components. It can be a date format, a price display, the appearance of an Angular component, etc.
+- A common configuration is defined in every library. The application's common configuration (__global configuration__) will be
+  the result of the __merge of all the common__ configurations.
 - The common configuration is __not overridden__ at component type (there will be no properties with the same name in
-  common config and components).
-- At runtime after the configuration of a component is computed it is extended by global config, so components can use
-  fields from global
+  common configuration and components).
+- At runtime, after the configuration of a component is computed, it is extended by the global configuration, so components can use
+  fields from the global configuration. (An example of this is available [below](#handle-global-config).)
 
-### Input config
+### Input configuration
 
-- The configuration of a __component instance__ can be overridden via @Input config
+- The configuration of a __component instance__ can be overridden via `@Input` configuration.
 
-__The priorities for the config__:
+### Priorities for the configuration
 
-- the highest priority is the one passed as input from a parent component
-- the second priority is the priority by component type set in the store
-- the lowest priority is the default config set on component (in the config.ts file of the component)
+- The highest priority is the one passed as input from a parent component
+- The second priority is the priority by component ID set dynamically
```

Comment:
I'm not sure to understand this part
also, is it not already kind of covered by the schema in the `Component configuration types` section ?

Reply:
This corresponds to the customized configuration, I can rephrase if the sentence is not clear enough
And yes, this is copied in the key takeaways section of the documentation